apache/zeppelin
# ZEPPELIN-2904 Show Remove Paragraph button upfront

### What is this PR for?
1. As part https://issues.apache.org/jira/browse/ZEPPELIN-2218, it was agreed upon that we can take out and move the remove paragraph button up front so that user don't have to go to the drop-down menu so that it will be a click away.
2. Paragraph control buttons are having a span as a markup and hence we had to add `cursor: pointer`. This was semantically incorrect and hence changed it to correct markup to use button and removed the unwanted inline css.
Due to this change, it will show mouse hover button highlight which was not present earlier.
3. The gear icon for the dropdown is used for interpreter bindings as well, instead changed the icon to horizontal ellipsis as it shows extra items which are not available upfront

### What type of PR is it?
[Improvement]

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-2834

### How should this be tested?
1. Build: mvn clean package -Denforcer.skip -DskipTests -Drat.skip
2. Open a paragraph
3. Check the menu item hover effect

### Screenshots (if appropriate)

Before
![remove-out-before](https://user-images.githubusercontent.com/1881135/30062809-1752482c-926a-11e7-8f60-c1d2987f7613.gif)

After
![remove-out-after](https://user-images.githubusercontent.com/1881135/30062813-194d8808-926a-11e7-8714-6aef2731f79c.gif)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
